exercises/concept/annalyns-infiltration/.docs/introduction.md

```diff
@@ -1,32 +1,13 @@
 # Introduction
 
+## Booleans
+
 The `Bool` type is one of the building blocks of the language.
 It aims at describing whether something is true or false, with the respective `True` and `False` values.
-The name `Bool` is short for Boolean, which is a field of mathematics related to logical thinking, named after the work of George Boole.
```

Comment:
I remember that at some point, there was a difference between the text visible in a concept page before starting the corresponding exercise and in the exercise introduction. So when it made sense, I tried to be more concise on the concept page and a bit more complete on the actual exercise introduction so that the student have enough relevant information for their exercise. I suppose that's why there is a difference in that concept exercise I wrote. Then I also added more info in the document available after finishing the concept (I don't remember the name of that doc).

Is this not the case anymore that there are two different purposes for these two documents? One just serves as a "concept teaser" and the other more complete for the actual exercise intro?

If those have been unified, then be careful about what you remove in this PR. It might remove useful info for students.

Reply:
Thanks for clearing that up for me. I also agree that it’s much easier to maintain the repo with the templates. As I don’t do much maintaining these day it’s probably better to align with your arguments. In that case, could you verify these intros that differ a lot and change the one used by the template to be enough introduction for students.